geoips/plugins/modules/output_checkers/image.py

```diff
@@ -118,7 +118,17 @@ def outputs_match(plugin, output_product, compare_product, threshold=0.05):
     )
     from PIL import Image
     import numpy as np
-    from pixelmatch.contrib.PIL import pixelmatch
+
+    try:
```

Comment:
We should wait on merging this PR until #438 is merged. Then, we should replace the optional import below with the new ``import_optional_dependencies`` functionality that has been created in #438.

Reply:
@biosafetylvl5 This comment is ready to be addressed.

Reply:
@evrose54 @jsolbrig The error message below has more details, and would be more useful to users. What benefit do we get by replacing with the `import_optional_dependencies` function? I'm so fine with doing so, but from what I see, it seems worse for the user. It also appears to fail to raise the error exception with traceback?

Let me know what I'm missing here 😵‍💫

Reply:
In this case I agree with you. Since you've implemented a better error message and this is a special case, I think we should keep this.

The history behind `import_optional_dependencies` is that there were dozens of these optional imports scattered all over the code, many with useless error messages. Maybe import_optional_dependencies wasn't the right solution, but it helped at least add something useful to those import errors to tell people what to do.

That said, the current error message still isn't very helpful and should be improved. We should think about how to improve the error message and how to get rid of as many optional imports as possible.

Reply:
Perhaps we can implement package specific messages, so it'd - for example - be applicable here, but still centralized. For now, I think this PR is gtg.